tests/suites/test_suite_psa_crypto_se_driver_hal_mocks.function

```diff
@@ -244,6 +255,7 @@ void mock_generate( )
     psa_key_handle_t handle = 0;
     psa_key_attributes_t attributes = PSA_KEY_ATTRIBUTES_INIT;
 
+    mock_generate_data.return_value = mock_generate_return_value;
```

Comment:
In commit "crypto_se_driver: add an error injection mechanism to the mocks", Should generate also have `mock_allocate_data.return_value = mock_alloc_return_value;` like import does?

Reply:
I've added this, along with a corresponding test.